tools/convert_datasets/mot/merge_coco_with_lvis.py

```diff
@@ -0,0 +1,99 @@
+import argparse
+import itertools
+import json
+import logging
+from pathlib import Path
+
+import numpy as np
+import pycocotools.mask as mask_util
+from pycocotools.coco import COCO
+from script_utils.common import common_setup
+from tqdm import tqdm
+
+ROOT = Path(__file__).resolve().parent.parent.parent
+
+
+def main():
+    # Use first line of file docstring as description if it exists.
+    parser = argparse.ArgumentParser(
+        description=__doc__.split('\n')[0] if __doc__ else '',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument(
+        '--lvis', type=Path, requirged=True, help='lvis json path')
+    parser.add_argument(
+        '--coco', type=Path, required=True, help='coco json path')
+    parser.add_argument(
+        '--mapping', type=Path, default=ROOT / 'data/lvis_coco_to_synset.json')
+    parser.add_argument('--output', type=Path, required=True)
+    parser.add_argument(
+        '--iou-thresh',
+        default=0.7,
+        type=float,
+        help=('If a COCO annotation overlaps with an LVIS annotations with '
+              'IoU over this threshold, we use only the LVIS annotation.'))
+
+    args = parser.parse_args()
+    args.output.parent.mkdir(exist_ok=True, parents=True)
+    common_setup(args.output.name + '.log', args.output.parent, args)
+
+    coco = COCO(args.coco)
+    lvis = COCO(args.lvis)
+
+    # transfer COCO category name LVIS according to synset
+    synset2lvis = {cat['synset']: cat['id'] for cat in lvis.cats.values()}
+    coco2lvis = {}
+    with open(args.mapping, 'r') as f:
+        mapping = json.load(f)
+    for cat in coco.cats.values():
+        mapped = mapping[cat['name']]
+        assert mapped['coco_cat_id'] == cat['id']
+        synset = mapped['synset']
+        if synset not in synset2lvis:
+            logging.debug(
+                f'Found no LVIS category for "{cat["name"]}" from COCO')
+            continue
+        coco2lvis[cat['id']] = synset2lvis[synset]
+
+    for img_id, _ in coco.imgs.items():
+        if img_id in lvis.imgs:
+            coco_name = coco.imgs[img_id]['file_name']
+            lvis_name = lvis.imgs[img_id]['file_name']
+            assert coco_name in lvis_name
+        else:
+            logging.info(f'Image {img_id} in COCO, but not annotated in LVIS')
+
+    lvis_highest_id = max(x['id'] for x in lvis.anns.values())
+    ann_id_generator = itertools.count(lvis_highest_id + 1)
```

Comment:
try `range`

Reply:
itertools.count is used to generate int after the highest id of lvis